lib/vectors.py

```diff
@@ -80,6 +80,13 @@ def __eq__(self, other_vec):
     def __str__(self):
         return str((self.x, self.y, self.z))
 
+    def distance(self, other):
+        diff = other - self
+        return sqrt(diff.x ** 2 + diff.y ** 2 + diff.z ** 2)
```

Comment:
Suggestion: `distance()` could be implemented as `return sqrt(self.distance2(other))`.